roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -67,7 +73,8 @@
       ansible.builtin.replace:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/clusters-app.yaml"
         regexp: "clusters-sub"
-        replace: "{{ czga_clusters_namespace }}"
+        replace: "{{ (czga_clusters_list | length == 1) | ternary(czga_clusters_list[0], ['']) }}"
+        after: "^kind: Application"
```

Comment:
This doesn't look like to be working. From https://www.distributed-ci.io/jobs/b444e94f-c12f-4306-b766-037af78a1866/jobStates?sort=date&task=d79548e4-484e-485a-9124-332b5e255149:
```
ok: [jumphost -> jumphost] => {"changed": false, "msg": "Pattern for before/after params did not match the given file: ^kind: Application(?P<subsection>.*)"}
```
Is this `after` field really needed?

Reply:
It seems that the beginning of line caret was at fault here, I didn't specify to treat the manifest as a multi-line, so only the very beginning of the manifest could count as the start of line. Removing the caret works. I also realized while testing that the empty string still was represented as an array, so I fixed that as well.

Reply:
Ack, let's see in the next tests if there are no issues in our side.